src/HuC3.md

```diff
@@ -0,0 +1,166 @@
+# HuC-3
+
+HuC-3 is an MBC used by some Game Boy games which, besides doing the
+usual MBC stuff, also provides a real-time clock, speaker, and infrared
+communication. The CR2025 coin cell is user-replaceable. It is a
+successor to the [HuC1](./HuC1.md).
+
+The HuC-3 is poorly understood. Observed behavior suggests the
+real-time clock and tone generator are implemented using a 4-bit
+microcontroller core with internal program ROM.
+
+## Memory
+
+### 0000–3FFF — ROM Bank 00 [read-only]
+
+Contains the first 16 KiB of the ROM.
+
+### 4000–7FFF — ROM Bank 00–7F [read-only]
+
+This area may contain any of the further 16 KiB banks of the ROM. It is
+unknown whether bank $00 can be selected here.
+
+### A000–BFFF — RAM Bank 00–03, or RTC/IR register [read/write]
+
+Depending on the current register selection and RAM Bank Number (see
+below), this memory space is used to access an 8 KiB external RAM Bank,
+or a single I/O Register.
+
+## Memory Control Registers
+
+### 0000–1FFF — RAM/RTC/IR Select [read/write]
+
+Writing to this register maps cart RAM, RTC registers or IR registers
+into memory at $A000–BFFF. Only the lower 4 bits are significant.
+
+Value | Feature
+------|-----------------------------
+  $0  | Cart RAM (read-only)
+  $A  | Cart RAM (read/write)
+  $B  | RTC command/argument (write)
+  $C  | RTC command/response (read)
+  $D  | RTC semaphore (read/write)
+  $E  | IR (read/write)
+
+If any other value is written, reads from $A000–BFFF return open bus
```

Comment:
Do we use "open bus values" anywhere else in the document? Or do we refer to them as rubbish etc? @ISSOtm

Reply:
Why be less explicit?  “Rubbish” could be interpreted as meaning the chip drives the bus with an unpredictable value.  “Open bus” describes what actually happens with the hardware.

Reply:
I'm not sure we use the term anywhere else, but yes, it's the correct term to use.